src/bun.js/bindings/UtilInspect.cpp

```diff
@@ -41,28 +42,26 @@ extern "C" JSC::EncodedJSValue JSC__JSValue__callCustomInspectFunction(
     JSC::EncodedJSValue encodedThisValue,
     unsigned depth,
     unsigned max_depth,
-    bool colors,
-    bool* is_exception)
+    bool colors)
 {
     JSValue functionToCall = JSValue::decode(encodedFunctionValue);
     JSValue thisValue = JSValue::decode(encodedThisValue);
     auto& vm = JSC::getVM(globalObject);
     auto scope = DECLARE_THROW_SCOPE(vm);
 
     JSObject* options = Bun::createInspectOptionsObject(vm, globalObject, max_depth, colors);
```

Comment:
does this also need to check for `!options` if `stylizeFn` was null and returned early?